src/db_impl.cc

```diff
@@ -470,6 +484,16 @@ Status TitanDBImpl::CreateColumnFamilies(
                  titan_table_factory[i]}));
       }
       blob_file_set_->AddColumnFamilies(column_families);
+      for (size_t i = 0; i < handles->size(); i++) {
+        auto rocks_cf_handle =
+            static_cast_with_check<rocksdb::ColumnFamilyHandleImpl>(
+                (*handles)[i]);
+        auto titan_handle = new TitanColumnFamilyHandle(
+            *rocks_cf_handle,
+            blob_file_set_->GetBlobStorage(rocks_cf_handle->GetID()).lock());
+        (*handles)[i] = titan_handle;
+        delete rocks_cf_handle;
```

Comment:
Question: is this rocks_cf_handle be used by TitanColumnFamilyHandle anymore? As it's used as the const ColumnFamilyHandleImpl& in TitanColumnFamilyHandle()